src/EditorFeatures/VisualBasic/NavigationBar/GenerateFinalizerItem.vb

```diff
@@ -48,6 +48,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.NavigationBar
                 modifiers:=New DeclarationModifiers(isOverride:=True),
                 returnType:=compilation.GetSpecialType(SpecialType.System_Void),
                 refKind:=RefKind.None,
+                isInitOnly:=False,
```

Comment:
this feels weird.  how do we do other accessors?  isn't this just a certain type of accessor, not a method?